src/pages/static/MarkdownReader.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { string } from 'prop-types';
+import remark from 'remark';
+import remark2react from 'remark-react';
+
+class MarkdownReader extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      text: '',
+    };
+  }
+
+  componentDidMount() {
+    fetch(this.props.content)
```

Comment:
Should we display a loading component while the content is loading? Maybe that can be a different task, as we don't have a loading component yet, we do have the design though .

Reply:
I agree that a "Loading" message would be good from a UX point of view, but we might not need a new component. 

Right now the state of `MarkdownReader` is initialised this way:

```
    this.state = {
      text: '',
    };
```

The value of `this.state.text` is then set in `componentDidMount`. We could initialise `this.state.text` to a "Loading..." message, and the message will be rendered by the component until the markdown loads. We could even use markdown syntax in that initial string!

```
    this.state = {
      text: '# Loading...',
    };
```

<img width="438" alt="screenshot 2018-11-30 at 19 56 50" src="https://user-images.githubusercontent.com/1347511/49311902-58859f00-f4da-11e8-967d-76aba1ded300.png">

Reply:
That's cool for now.